contracts/DebtKernel.sol

```diff
@@ -76,6 +76,8 @@ contract DebtKernel is Pausable {
     mapping (bytes32 => bool) public issuanceCancelled;
     mapping (bytes32 => bool) public debtOrderCancelled;
 
+    mapping (address => address) public debtorConsentDelegation;
```

Comment:
Feels odd to just do this for debtors and not for other actors in the protocol — in particular, I think there are many reasons this could be useful for creditors as well.  Would pretty dramatically simplify the way creditor proxies work, for instance.